docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -137,17 +152,55 @@ Relogin to fetch updated certificates:
 $ tsh logout; tsh login --proxy=example.com --user=alice
 ```
 
-Having the necessary permissions, let's add a device to the inventory. Replace
-the `$SERIAL` below with your macOS device serial number (visible under Apple ->
-"About This Mac" -> "Serial number", or use the command below).
+Before we can enroll the device, we need to register it. In order to register
+a device, we need to first determine its serial number.
+
+<Tabs>
+  <TabItem label="macOS">
+    The serial number of a macOS device can be determined in two ways.
+
+    Through the UI, visible under Apple -> "About This Mac" -> "Serial number".
+
+    Through the terminal:
+    ```code
+    $ ioreg -c IOPlatformExpertDevice -d 2 | grep -i IOPlatformSerialNumber | awk -F'"' '{print $4}'
+    (=devicetrust.asset_tag=)
+    ```
+  </TabItem>
+
+  <TabItem label="Windows">
+    Determining the serial number for a device running Windows is a little more
+    complicated than on macOS. This is because Windows devices can have multiple
+    serial numbers depending on the configuration made by the manufacturer.
+
+    Teleport will pick the first available value from the following:
+
+    - System asset tag
+    - System serial number
+    - Baseboard serial number
+
+    To find the value chosen by Teleport, run the following command:
+
+    ```code
+    $ tsh device collect
+    DeviceCollectedData {
+      ...
+      "serial_number": "(=devicetrust.asset_tag=)",
+      ...
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+Replace the `$SERIAL` below with the serial number obtained from the device you
+wish to enroll and `$OS` with `macos` or `windows`, and execute the command:
 
 ```code
-$ SERIAL="$(ioreg -c IOPlatformExpertDevice -d 2 | grep -i IOPlatformSerialNumber | awk -F'"' '{print $4}')"
-$ tctl devices add --os=macos --asset-tag="$SERIAL"
+$ tctl devices add --os=$OS --asset-tag="$SERIAL"
```

Comment:
```suggestion
$ export OS=macos
$ export SERIAL='CHANGEME'
$ tctl devices add --os="$OS" --asset-tag="$SERIAL"
```

SERIAL could be `export SERIAL="$(tsh device serial-number)"`, per comment above.

Reply:
It feels a little confusing to have commands in the same code block that need to be run on the admin machine and also on the machine to be enrolled.

Reply:
420fd9a1af